docs/integrations/microsoft-azure/azure-app-service-plan.md

```diff
@@ -14,7 +14,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 For Azure App Service Plan, you can collect the following metrics:
 
-* **Platform Metrics for Azure App Service Plan**. These metrics are available in [Microsoft.Web/serverfarms](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-web-serverfarms-metrics) namespace. For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/app-service/web-sites-monitor#understand-metrics). App Service plan metrics are available only for plans in Basic, Standard, and Premium tiers.
```

Comment:
Also need to add a note that this is supported for Dedicated plans (basic, standard, premium etc) only along with the doc link

Reply:
Done

Reply:
Basic, Standard, Premium  and Isolated tiers

Reply:
Its there on line #19